src/mcp-sdk/src/Capability/Tool/MetadataInterface.php

```diff
@@ -24,6 +33,42 @@ public function getDescription(): string;
      *       description?: string,
      *   }>,
      * }
+     *
+     * @see{https://modelcontextprotocol.io/specification/2025-06-18/schema#tool-inputschema}
      */
     public function getInputSchema(): array;
+
+    /**
+     * @return array{
+     *   type?: string,
```

Comment:
wondering if this should be `type: 'object'` - at least when looking at the docs here:
https://modelcontextprotocol.io/specification/2025-06-18/schema#tool